Automattic/wp-calypso
# Premium Squared: Allow elligible premium themes to be activated (take two)

Currently, themes included in the Premium Squared bundle can't be activated from the Calypso theme showcase. This PR adds the necessary bits of code to allow theme activation of Premium Squared themes (e.g. Automattic + WooThemes themes).

**Note:** This PR is actually a fix of #11112, which was reverted on #11178 because of errors.

The code is mostly the same as #11112, but with a minor fix introduced in 2a94588365696eb34c4c2d8c842183bf86d1259b, which gets rid of the `FEATURE_PREMIUM_SQUARED` constant from `client/state/lib/plans/constants.js`, and instead checks for the Premium Plan directly via `getSitePlanSlug`.

- - - 

**Testing Instructions**

On a site with the Premium Upgrade, you should be able to install Automattic and WooThemes Premium themes without having to purchase them.

Here are the themes you should be able to activate:
- Automattic: https://theme.wordpress.com/?theme_shop=automattic
- WooThemes: https://theme.wordpress.com/?theme_shop=woothemes
